src/sql/parts/dashboard/tabContents/webviewTabContent.component.ts

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the Source EULA. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import 'vs/css!./webviewTabContent';
+
+import { Component, forwardRef, Input, OnInit, Inject, ChangeDetectorRef, ElementRef } from '@angular/core';
+
+import Event, { Emitter } from 'vs/base/common/event';
+import Webview from 'vs/workbench/parts/html/browser/webview';
+import { Parts } from 'vs/workbench/services/part/common/partService';
+import { IDisposable } from 'vs/base/common/lifecycle';
+
+import { DashboardTab } from 'sql/parts/dashboard/common/interfaces';
+import { TabConfig } from 'sql/parts/dashboard/common/dashboardWidget';
+import { DashboardServiceInterface } from 'sql/parts/dashboard/services/dashboardServiceInterface.service';
+import { IDashboardWebview } from 'sql/services/dashboardWebview/common/dashboardWebviewService';
+
+import * as data from 'data';
+import { memoize } from 'vs/base/common/decorators';
+
+@Component({
+	template: '',
+	selector: 'webview-tab-content'
```

Comment:
I wouldn't call this tab content, cause it can be a content to other things than just tab